world_02.py

```diff
@@ -3,100 +3,125 @@
 import pygame
 
 
+class Player():
+
+    TURN_DEG = 30
+    STEP = 1
+    PLAYER_RADIUS = 5
+
+    def __init__(self, world, coordinates):
```

Comment:
I wanted the player to have access to information about the world she is in. Not sure if this is the right way to do that.